docs/repositories-configure/using-submodules.md

```diff
@@ -2,39 +2,53 @@
 
 [Git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) allow you to keep a Git repository as a subdirectory within another Git repository. Git submodules are helpful in maintaining a shared configuration file for your team, and then applying it to multiple Git repositories.
 
-## Enabling submodules
+By default, Codacy does normal Git clones that **don't include submodules** to ensure that we only clone necessary repositories. If your organization needs to use submodules, you can request Codacy to enable this feature for you.
 
-By default, Codacy does normal Git clones that don't include submodules to ensure that we only clone necessary repositories. However, if your organization needs to use submodules, please contact us at <mailto:support@codacy.com> asking to enable submodules on Codacy.
+## Prerequisites for using submodules
 
-After we enabled submodules for your organization, do the following:
+1.  Contact us at <mailto:support@codacy.com> asking to enable submodules on Codacy.
 
-1.  **If you're using Codacy Self-hosted**, you must [update the license](../chart/maintenance/license.md).
+1.  **If you're using Codacy Self-hosted**, [update your license](../chart/maintenance/license.md).
 
 1.  If your submodules are:
-
-    -   **Public repositories**, make sure that your Git URL uses the HTTPS protocol
+    -   **Public repositories**, make sure that your Git URL uses the HTTPS protocol.
     -   **Private repositories**, make sure that your Git URL uses the SSH protocol.
 
-1.  If you already have repositories on Codacy where you want to use submodules, navigate to the page **Settings** and click the button **Generate New User Key** for each of these repositories.
+## Enabling submodules on a repository {: id="enable"}
+
+When using submodules, you must do the following for all your existing and new repositories:
+
+1.  Open the repository **Settings**, tab **General**. In the **Danger zone** area, you have the **SSH Key** generated by Codacy to access your repository.
+
+    !!! note
+        Codacy generates this repository key when you add a repository to Codacy and uses it to clone that repository. When you're using submodules, Codacy needs to clone additional repositories it may not have access to. To overcome this, Codacy must use an SSH key of your user account to have access to the same repositories as your user.
```

Comment:
Since this information is now tightly coupled with the step of identifying the SSH key, I suggest that we drop the admonition and simply have the information as a bare paragraph supporting the instruction.